app/models/hackathon/regional.rb

```diff
@@ -10,7 +10,7 @@ module Hackathon::Regional
         hackathon.province = result.province || result.state
         hackathon.city = result.city
         hackathon.address = result.address
-        hackathon.street = [result.house_number, result.street].compact.join(" ")
+        hackathon.street = [result.try(:house_number), result.try(:street)].compact.join(" ").presence
```

Comment:
i got no method errors from the stubs

Reply:
Fixed by [`d554c0f` (#73)](https://github.com/hackclub/hackathons-backend/pull/73/commits/d554c0fcf08bc3996e2ba5fb8312f434c41eae2f)